src/ethereum/transaction/transaction.service.ts

```diff
@@ -17,15 +18,24 @@ export class TransactionService {
     @InjectMetric(METRIC_TRANSACTION_COUNTER)
     private transactionCount: Gauge<string>,
 
+    private providerService: ProviderService,
     private configService: ConfigService,
-  ) {
-    this.transactionCount.set({ status: TransactionStatus.error }, 0);
-    this.transactionCount.set({ status: TransactionStatus.pending }, 0);
-    this.transactionCount.set({ status: TransactionStatus.confirmed }, 0);
-    this.transactionCount.set({ status: TransactionStatus.timeout }, 0);
```

Comment:
why are we getting rid of it here? This is important, because `increase()` will not catch null -> 1 but will catch 0 -> 1

Reply:
It was moved to `onModuleInit` method body. Anyway, it's reverted now.